src/lib/components/workspace/invite-to-workspace/InviteToWorkspace.svelte

```diff
@@ -0,0 +1,223 @@
+    addUsersInWorkspacePayload,
+  } from "$lib/utils/dto";
+  import { notifications } from "$lib/utils/notifications";
+  import { WorkspaceRole } from "$lib/utils/enums";
+  import { createDynamicComponents } from "$lib/utils/helpers/common.helper";
+  const emailstoBeSentArr: string[] = [];
+
+  export let addUsersInWorkspace: (
+    id: string,
+    data: addUsersInWorkspacePayload,
+  ) => Promise<any>;
+  export let addUsersInWorkspaceInRxDB: (
+    id: string,
+    data: addUsersInWorkspace[],
+  ) => Promise<void>;
+  export let currentWorkspaceDetails: { id: string; name: string };
+  let showErrors = false;
+  let defaultRole = "select";
+  let selectedRole = defaultRole;
+  let currentEmailEntered: string;
+
+  function removeElement(event: Event): void {
+    const id = event.target?.id;
+    const removeElement = document.getElementById(id) as HTMLElement;
+    const emailContainer = document.getElementById(
+      "input-email",
+    ) as HTMLElement;
+    emailContainer.removeChild(removeElement);
+  }
+
+  const handleEmailOnAdd = (email: string) => {
+    email = email.replace(",", "");
+    email = email.trim();
+    emailstoBeSentArr.push(email);
+    const emailDiv: HTMLElement = createDynamicComponents(
+      "div",
+      "d-flex bg-emailInviteBackgroundColor gx-1 px-1 justify-content-center rounded-1 align-items-center",
+    );
+    const emailContentSpan = createDynamicComponents("span", "");
+    const closeIconBtn = createDynamicComponents("img", "bg-transparent", [
+      { eventType: "click", eventHandler: removeElement },
+      {
+        eventType: "mouseleave",
+        eventHandler: () => {
+          closeIconBtn.src = closeIcon;
+        },
+      },
+      {
+        eventType: "mouseenter",
+        eventHandler: () => {
+          closeIconBtn.src = closeIconWhite;
+        },
+      },
+    ]) as HTMLImageElement;
+    emailDiv.id = email;
+    closeIconBtn.id = email;
+    closeIconBtn.src = closeIcon;
+    emailContentSpan.innerHTML = email;
+    emailDiv.appendChild(emailContentSpan);
+    emailDiv.appendChild(closeIconBtn);
+    const emailContainer: HTMLElement = document.getElementById(
+      "input-email",
+    ) as HTMLElement;
+    emailContainer.appendChild(emailDiv);
+    currentEmailEntered = "";
+  };
+  const handleInvite = async () => {
+    showErrors = true;
+    const data: addUsersInWorkspacePayload = {
+      users: emailstoBeSentArr,
+      role: selectedRole,
+    };
+    if (emailstoBeSentArr && emailstoBeSentArr.length > 0 && selectedRole) {
+      const response = await addUsersInWorkspace(
+        currentWorkspaceDetails.id,
+        data,
+      );
+      if (response && response.data.data) {
+        const newTeam: addUsersInWorkspace[] = response.data.data.users;
+        addUsersInWorkspaceInRxDB(currentWorkspaceDetails.id, newTeam);
+        notifications.success(
+          `Invite Sent to ${emailstoBeSentArr.length} for ${currentWorkspaceDetails.name}`,
```

Comment:
Invites sent to*

Reply:
This as per figma, will connect with design team for updations.